DPPP/GainCalAlgorithm.cc

```diff
@@ -147,7 +147,7 @@ namespace DP3 {
         double ginit=0.;
         bool ginitcomputed=false;
         for (unsigned int ant=0; ant<_nUn; ++ant) {
-          if (!isFinite(_g(ant,0).real()) ) {
+          if (!std::isfinite(_g(ant,0).real()) ) {
```

Comment:
In implementation files, I think using `using namespace std::` is fine as well.

Reply:
I don't object to a `using namespace`, but there are some overloads inside casacore with the same name as things in std (e.g. sqrt, cos, etc). In those cases it is better to use std:: in front of them. Therefore, as a rule I'd rather use an explicit std::, but if it gets crazy using namespace is fine (unless of course it is used in templates and the chosen overload depends on the type).

Reply:
Btw 'abs' always needs std:: in front for floating point; if that's not done, the compiler may select int abs(int), and truncates the float.